### PR TITLE
Show power usage of the OSR in the main README

### PR DESCRIPTION
**Pull request by Achllle**

### CODE REVIEW

**Comment by abust005** [comment]
Given that we recommend a 4S LiPo battery, might this be the adequate place to add a small warning about not discharging your LiPo battery too far? While the dropout regulator does need more than 12V, the battery can still be drained to levels that are detrimental to battery health

File: README.md [hunk 1]
```diff
@@ -48,6 +48,10 @@ This rover is designed to function similarly to the 6 wheel rover designs on Mar
 
 A Raspberry Pi acts as the "brain" of this rover for its versatility, accessibility, simplicity, and ability to add and upgrade your own modifications. Any method with which you can communicate with a Raspberry Pi (bluetooth, WiFi, USB devices, etc) can be interfaced into the control system of the robot.
 
+For a 7.2Ah battery and with some driving on rough terrain, expect to get at least 3 hours of drive time. Below chart was recorded while driving on rocky terrain and plots voltage, current draw, and commanded velocity over time. When the voltage drops below 13-14V, the rover will power down. You can increase drive time by getting a bigger battery and/or by having the rover carry a second battery that can be swapped out.
```
Comment (Achllle) [inline README.md:51]:
```suggestion
For a 7.2Ah battery and with some driving on rough terrain, expect to get at least 3 hours of drive time. Below chart was recorded while driving on rocky terrain and plots voltage, current draw, and commanded velocity over time. When the voltage drops below 13-14V, the rover will power down. You can increase drive time by getting a bigger battery and/or by having the rover carry a second battery that can be swapped out.

> *WARNING*: Be careful to not discharge a LiPo battery too far as that drastically lowers the lifetime of your battery and can potentially make it hard or impossible to recharge.
```

**Comment by Achllle** [comment]
good point, added that warning in!

**Comment by abust005** [comment]
Looks good!

**Pull request merged**